src/zui/ZUIStatusChip/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import { makeStyles } from '@mui/styles';
+import { Box, Theme, Typography } from '@mui/material';
+
+import { getContrastColor } from 'utils/colorUtils';
+
+const useStyles = makeStyles<Theme, { status: ActivityStatus }>((theme) => ({
+  chip: {
+    alignItems: 'center',
+    backgroundColor: ({ status }) => theme.palette.activityStatusColors[status],
+    borderRadius: '2em',
+    color: ({ status }) =>
+      getContrastColor(theme.palette.activityStatusColors[status]),
+    display: 'inline-flex',
+    padding: '0.438rem 0.625rem',
+  },
+}));
+
+type ActivityStatus =
+  | 'cancelled'
+  | 'draft'
+  | 'expired'
+  | 'published'
+  | 'scheduled';
+
+interface ZUIStatusChipProps {
+  label: string;
```

Comment:
Would it maybe make sense to not have the label as a property, but instead just derive it from the status? I know our labels differ a little bit (e.g. "expired" and "unpublished") but in those cases I think we should either try to consolidate, or if there are situations where we really can't perhaps we should have two separate statuses?

Reply:
I toyed with the idea but this is its own nest of angry wasps, that i would love to discuss. there are many statuses, and what the label of a status is, is different across features...